helm/csi-driver-lvm/templates/csi-lvm-driverinfo.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: storage.k8s.io/v1beta1
+apiVersion: storage.k8s.io/v1
```

Comment:
Might be better to check the version of kubernetes and use a conditional? Not sure if the maintainers want backwards compatibility for k8s less than 1.22 🤔

Reply:
With the current chart I think it's required to have at least Kubernetes v1.19. So maybe I can add this to the README.

Reply:
Okay, so the conditional is already in there and I verified it's working even with K8s 1.16. It's also upstream now. :)